basics_dev/coding-style.md

```diff
@@ -165,7 +165,7 @@ Security coding guidelines
        height = untrusted_conf.height;
     ~~~
 
--   Use another variables, without the `untrusted_` prefix to hold the sanitized values, as shown above.
```

Comment:
not quite right either, because one untrusted_* input may be parsed into multiple trusted values